Tools/_sunrise/test_pr_check_summary.py

```
@@ -0,0 +1,586 @@
+            "workflow_run": {
+                "event": "pull_request",
+                "head_sha": "oldsha",
+                "pull_requests": [{"number": 42}],
+            }
+        }
+
+        pull_request = resolve_pull_request(client, payload, "workflow_run", explicit_pr_number=None)
+
+        self.assertIsNone(pull_request)
+
+    def test_download_job_log_uses_default_urlopen_and_extracts_zip_logs(self):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `test_download_job_log_uses_default_urlopen_and_extracts_zip_logs`

Add return type annotation: `None`

(ANN202)